tests/libmemcached-1.0/dynamic_mode_test.cc

```diff
@@ -436,7 +436,7 @@ test_return_t polling_test(memcached_st *ptr)
 
 #include "tests/libmemcached-1.0/dynamic_mode_test.h"
 
-void get_world(libtest::Framework *world)
+void get_world_dynamic_mode(libtest::Framework *world)
```

Comment:
This was causing compilation issues and duplicate error. See note on my PR #11. I ended up leaving dynamic_mode_test alone and modified the all_tests. These two files were causing duplicate function errors during compilation.